source/kraken/realtime.cpp

```diff
@@ -496,6 +584,7 @@ create_disruption(const std::string& id,
         } else {
```

Comment:
Just a reminder here because it's an "outdated" comment.
Above:
I think you still have to manage `trip_effect = nt::disruption::Effect::ADDITIONAL_SERVICE;` (it's not managed from get_calculated_trip_effect(stops) ), no?
We should see it in test, maybe.

Reply:
ADDTIONAL_SERVICE is a "main effect". It is manage here
https://github.com/CanalTP/navitia/pull/2689/files#diff-a1c1532aa9f44bfd3b8a77e208515e23R589